docs/reference/manifest/progid.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ProgID element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# ProgID element
+
+Specifies the programmatic identifier of the equivalent COM add-in for the task pane of your web add-in.
+
+**Add-in type:** Task pane
+
+## Syntax
+
+```XML
+    <ProgID>{progid}</ProgID>  
+```
+
+## Contained in
+
+[EquivalentAdd-in](equivalentaddin.md)
+
+## Remarks
+
+You must specify the programmatic identifier of the COM add-in that contains equivalent UI for your web add-in's task pane UI.
```

Comment:
Is it worth calling out how to find the ProgID of a COM add-in?

Reply:
There's lots of online docs on COM if someone forgets, plus a few different ways that might make this cumbersome to document here.